keyboards/handwired/jscotto/scottoalp/info.json

```diff
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "Joe Scotto",
+    "keyboard_name": "ScottoAlp",
+    "maintainer": "joe-scotto",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        //  4, 5, 6, 7, 8, 9, A3, A2, A1, A0
```

Comment:
Comments should be removed.

Reply:
Comment is to clearly show which pins to wire up the board since it's handwired.